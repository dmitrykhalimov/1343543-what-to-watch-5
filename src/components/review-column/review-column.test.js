import React from "react";
import renderer from "react-test-renderer";
import ReviewColumn from "./review-column";

const MockComponent = () => {

  return (
    <div>
    </div>
  );
};

describe(`ReviewColumn`, () => {
  it(`Should ReviewColumn render correctly`, () => {
    const tree = renderer
      .create(
          <ReviewColumn>
            <MockComponent/>
          </ReviewColumn>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
