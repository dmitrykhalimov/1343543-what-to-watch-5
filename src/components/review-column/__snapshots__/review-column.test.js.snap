// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewColumn Should ReviewColumn render correctly 1`] = `
<div
  className="movie-card__reviews-col"
>
  <div />
</div>
`;
